src/app/app.component.html

```diff
@@ -1,7 +1,2 @@
-<div class="container-fluid">
-  <div class="row">
-    <div class="col">
-      <h2>this is a tester test</h2>
-    </div>
-  </div>
-</div>
+<app-navbar></app-navbar>
+<h2>a;lskfj;alkdjsf</h2>
```

Comment:
Looks pretty good, but is this a typo? Didn't you mean a;lskfj;alk**i**jsf instead? Otherwise should work properly